assets/js/controllers/send.controller.js

```diff
@@ -32,14 +33,25 @@ function SendCtrl($scope, $log, Wallet, Alerts, currency, $uibModalInstance, $ti
     sweepAmount: null,
     destinations: [null],
     amounts: [null],
-    fee: Wallet.settings.feePerKB,
+    fee: 10000,
     note: "",
     publicNote: false
   };
 
-  $scope.payment = new Wallet.payment();
+  $scope.payment = new Wallet.payment({ feePerKb: 30000 });
```

Comment:
30 satoshi per byte is a reasonable fee setting, it's there in case the dynamic fee service fails in some way.